templates/candlepin.container.epp

```diff
@@ -0,0 +1,27 @@
+<%- |
+  String[1] $image,
+| -%>
+### File managed with puppet ###
+## Module: '<%= $module_name %>'
+[Unit]
+Description=Candlepin
+After=local-fs.target
+
+[Install]
+WantedBy=multi-user.target default.target
+
+[Container]
+Image=<%= $image %>
+PodmanArgs=--cgroups=enabled
+LogDriver=journald
+Network=host
+
+Volume=/etc/tomcat/server.xml:/etc/tomcat/server.xml:Z
+Volume=/etc/tomcat/login.config:/etc/tomcat/login.config:Z
+Volume=/etc/tomcat/cert-roles.properties:/etc/tomcat/cert-roles.properties:Z
+Volume=/etc/tomcat/cert-users.properties:/etc/tomcat/cert-users.properties:Z
+Volume=/etc/tomcat/conf.d/jaas.conf:/etc/tomcat/conf.d/jaas.conf:Z
+Volume=/etc/tomcat/tomcat.conf:/etc/tomcat/tomcat.conf:Z
+Volume=/etc/candlepin:/etc/candlepin:Z
+Volume=/var/log/candlepin:/var/log/candlepin:Z
+Volume=/var/log/tomcat:/var/log/tomcat:Z
```

Comment:
In my testing of a completely unrelated application I didn't need `:Z`, but I was using plain volumes. Is this really needed?

Reply:
I think it depends on the application, the path, if it's read-only or read-write needed and SELinux policy as I understand it. 

These two have some useful info:

- https://github.com/containers/podman/blob/main/troubleshooting.md
- https://docs.podman.io/en/latest/markdown/podman-run.1.html